### PR TITLE
validate registration codes the right way

### PR DESCRIPTION
**Pull request by pdsarin**
Also hacks the simulator to be able to run any of the 7 drills.

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -51,6 +51,11 @@ functions:
 
   handleCommand:
     handler: command_stream.handle_command
+    environment:
+      DIALOG_TABLE_NAME_SUFFIX: ${self:provider.stage}
+      REGISTRATION_VALIDATION_URL: https://eslworks-api-production.herokuapp.com/slowcovid/lookup
+      REGISTRATION_VALIDATION_KEY: ${env:REGISTRATION_VALIDATION_KEY}
```
Comment (jeffreysilver) [inline serverless.yml]: We're already storing the auth_username and auth_password in the .env files and using them in `clients/eslworks.py`. I'll make a ticket to clean that up and use the validation key envvar instead

Comment (pdsarin) [reply serverless.yml]: Are we talking about the same thing? This is a special validation key for Kai's API.

Comment (jeffreysilver) [reply serverless.yml]: Yes, we are. We currently hit Kai's api in the registration lambda

**Comment by pdsarin** [comment]
ah, ok. Thanks for cleaning up.

**Pull request merged**